src/openqaoa-qiskit/README.md

```diff
@@ -38,35 +55,46 @@ pip install openqaoa-qiskit
 ```
 
 ### Installation instructions for Developers
-OpenQAOA-Qiskit does not yet support developer install as a standalone package. If you wish to work in developer mode, please install the entire library. Instructions are available [here]()
 
-Should you face any issue during the installation, please drop us an email at openqaoa@entropicalabs.com or open an issue!
+OpenQAOA-Qiskit does not yet support developer install as a standalone package.
+If you wish to work in developer mode, please install the entire library.
+Instructions are available [here]()
```

Comment:
Can you please add the missing link? https://github.com/entropicalabs/openqaoa/blob/dev/README.md